sys/dev/netmap/netmap_kern.h

```diff
@@ -1666,16 +1666,24 @@ netmap_load_map(struct netmap_adapter *na,
 
 static inline void
 netmap_unload_map(struct netmap_adapter *na,
-	bus_dma_tag_t tag, bus_dmamap_t map)
+	bus_dma_tag_t tag, bus_dmamap_t map, u_int sz)
 {
-	u_int sz = NETMAP_BUF_SIZE(na);
-
 	if (*map) {
 		dma_unmap_single(na->pdev, *map, sz,
 				 DMA_BIDIRECTIONAL);
 	}
 }
 
+static inline void
+netmap_sync_map(struct netmap_adapter *na,
```

Comment:
In anticipation for the TX path changes, may you please pass the direction as an additional ```enum txrx``` parameter?

Reply:
I've added the parameter, but left function body untouched, 'cause I'm a bit confused about what dma_sync_single_for_*() call and DMA_* direction to use in case of NR_TX ring.